examples/with-docker/Dockerfile

```diff
@@ -37,6 +37,8 @@ WORKDIR /app
 ENV NODE_ENV production
 # Uncomment the following line in case you want to disable telemetry during runtime.
 # ENV NEXT_TELEMETRY_DISABLED 1
+# Uncomment the following line in case you use sharp library
+# ENV NEXT_SHARP_PATH /app/node_modules/sharp
```

Comment:
Hi, this shouldn't be needed unless the sharp path is being installed in a separate docker layer, if it's installed as a dependency on the project it should work fine without the env variable.

Reply:
Thank you for your comment!
When I deploy to AWS Copilot, It doesn't work sharp. I agree to not need env variable comment but We need to check It dose work sharp when Deploy to docker with standalone.

Sorry, My english is not well

Reply:
If `sharp` is installed in the project it should pick it up correctly, are you customizing how the dependencies are installed?

Reply:
I try to docker locally. It did work.
I think It's problem of AWS Copilot

Here's my setting

https://github.com/insik-han/next-ts/blob/main/package.json